pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
```

Comment:
maybe revert this whole file for this PR?

Reply:
good call -- do you know why the tests here are failing though? Will be important for the gcs pr.